vpr/src/base/vpr_api.cpp

```diff
@@ -1183,6 +1188,30 @@ bool vpr_analysis_flow(t_vpr_setup& vpr_setup, const t_arch& Arch, const RouteSt
         VTR_LOG("*****************************************************************************************\n");
     }
 
+    /* If routing is successful, apply post-routing annotations
+     * - apply logic block pin fix-up
+     *
+     * Note: 
+     *   - Turn on verbose output when users require verbose output
+     *     for packer (default verbosity is set to 2 for compact logs)
+     */
+    if (route_status.success()) {
```

Comment:
Per the other comment, this block should be part of `vpr_route_flow`?

Reply:
I remember that we have discussed where to call this API during the last meeting. The conclusion is to add it to the beginning of analyze flow, so that analyze flow can always output a correct final result. Otherwise, when users call analyzing flow only, the .net and .routing files will not be synchronized as the mismatches are not stored anywhere.

Reply:
I was suggesting it be at both

Reply:
I'm OK with just outputting it in analysis if that's easier, for the reasons we discussed in the meeting. I'm not too worried about it not being output after routing; the common flow if you don't want to specify many options is the whole flow is run anyway and you're get this .net file in that case.